### PR TITLE
new pages for String.prototype.concat and String.prototype.indexOf

### PR DESCRIPTION
**Pull request by faizaanceg**
Closes FreeCodeCamp/FreeCodeCamp#3430 
Closes FreeCodeCamp/FreeCodeCamp#3429 

Added 2 new pages for String.prototype.concat and String.prototype.indexOf

If these are okay, then I would like to add rest of the String prototypes in this branch itself.

### CODE REVIEW

**Comment by SaintPeter** [comment]
Note that you can reference/close the open issues on the main FCC repo:
https://github.com/blog/1439-closing-issues-across-repositories

Update your submision text to say:
Closes FreeCodeCamp/FreeCodeCamp/issues/3430
Closes FreeCodeCamp/FreeCodeCamp/issues/3429

This will auto-close these issues when the pull request is accepted.

File: js-String-prototype-concat.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+The concat() method combines the text of two or more strings and returns a new string.
+
+**Syntax**
+```js
+str.concat(string2, string3[,..., stringN]);
+```
+
+### Parameters
+
+**string2...string*N***
```
Comment (bugron) [inline js-String-prototype-concat.md]: I think this should be `**string2...stringN**` or something like that.

Comment (faizaanceg) [inline js-String-prototype-concat.md]: Well I was following the MDN docs and they had italicized the N, so I just went with that. It rendered correctly in my editor.

Comment (SaintPeter) [inline js-String-prototype-concat.md]: The line should read:
`str.concat(string2 [,..., stringN]);`

Comment (faizaanceg) [inline js-String-prototype-concat.md]: Hmm, ok. Will change it now

**Pull request merged**